test/e2e/specs/all-features-confirmation/navbar/no-logo.spec.js

```diff
@@ -8,8 +8,8 @@ describe('Navbar ', function() {
         browser.ignoreSynchronization=true;
         browser.get(browser.params.url + "/recordset/#" + browser.params.catalogId + "/product-navbar:accommodation");
         navbar = element(by.id('mainnav'));
-        menu = element(by.id('navbar-menu'));
-        loginMenu = element(by.css('.chaise-login-menu'));
+        menu = element(by.css('.navbar-menu-options'));
+        loginMenu = element(by.css('.login-menu-options'));
```

Comment:
This selector was selecting the `ul` element that contained all the login dropdown options. This selector should mimic that still and be changed to `element.all(by.css('.username-display > .dropdown-menu))` so we are selecting that same element.